generics/constants/api-responses.js

```diff
@@ -121,5 +121,6 @@ module.exports = {
     "TEMPLATE_TASK_NOT_UPDATED": "Could not updated template task",
     "PROJECT_TEMPLATE_UPDATED" : "Successfully updated project template",
     "PROJECT_TEMPLATE_TASK_UPDATED" : "Successfully updated project template task",
-
+    "PROJECT_TEMPLATE_FETCH_ERROR" : "TemplateId or Link either one is required",
+    "GOT_SOLUTIONS_NO_TEMPLATE_ID" : "Solution fetched successfully. But projectTemplateId is empty"
```

Comment:
Change the text
"TEMPLATE_ID_NOT_FOUND_IN_SOLUTION" : "Could not found templateId in solution"